cloudinit/config/cc_ssh.py

```diff
@@ -9,11 +9,13 @@
 """
 SSH
 ---
-**Summary:** configure ssh and ssh keys
+**Summary:** configure ssh and ssh keys (host and authorized)
 
-This module handles most configuration for ssh and ssh keys. Many images have
-default ssh keys, which can be removed using ``ssh_deletekeys``. Since removing
-default keys is usually the desired behavior this option is enabled by default.
+This module handles most configuration for ssh and both host and authorized ssh
+keys. Many images have default ssh keys, which can be removed using
```

Comment:
```suggestion
keys. Many images have default host ssh keys, which can be removed using
```

Looking at it, perhaps "Many images" should be the start of a new paragraph, as it doesn't apply to authorized keys at all.